documentation/installation.md

```diff
@@ -49,16 +60,24 @@ command:
 
     vagrant ssh
 
-To check if everything is set up properly you can run ``make test.unit`` inside
+Inside the virtual machine, if you are setting up a development environment,
+set HUEY_ALWAYS_EAGER to false in /home/vagrant/opencraft/.env. Also, create
+a superuser account which will be used to log in to Ocim:
+
+    make manage createsuperuser
+
+To check if everything is set up properly, you can run ``make test.unit`` inside
 your new environment.
 
 Vagrant will set up a VirtualBox share mapping your local development directory
 to `/vagrant` inside the virtual machine. Any changes you make locally will be
 reflected inside the virtual machine automatically.
 
 Vagrant will map port 5000 inside the virtual machine to port 5000 on the host.
-Once you have set everything up you will be able to access the development
-server at http://localhost:5000/ using your web browser.
+Once you have set everything up, you will be able to access the home webpage of
+the development server at http://localhost:5000/ using your web browser. Log in
+with the previously created superuser account credentials and you will then be
+redirected to the Ocim webpage which lists the OpenEDX instances.
```

Comment:
```suggestion
redirected to the Ocim webpage which lists the Open edX instances.
```

Reply:
Again this is a pretty odd case, but I'll just be consistent and fix it.